warp10/src/main/java/io/warp10/hadoop/WritableUtils.java

```diff
@@ -50,30 +51,30 @@
  * Utility class to convert to/from Writables
  */
 public class WritableUtils {
-  
+
   private static final boolean strictWritables;
   private static final boolean rawWritables;
-  
+
   static {
     if ("true".equals(WarpConfig.getProperty(Configuration.CONFIG_WARPSCRIPT_HADOOP_STRICTWRITABLES))) {
```

Comment:
Could be replaced by: `strictWritables = "true".equals(WarpConfig.getProperty(Configuration.CONFIG_WARPSCRIPT_HADOOP_STRICTWRITABLES));`